apps/frontend/src/app/components/Header/index.js

```diff
@@ -39,12 +40,14 @@ const toggler = [
 ];
 
 function Header({ name, subName, onPress }) {
+  const posthog = usePostHog();
   const local_info = JSON.parse(localStorage.getItem("user-info"));
   const navigate = useNavigate();
 
   useEffect(() => window.scrollTo(0, 0));
 
   function logout() {
+    posthog.capture("User Signed Out",{user:local_info});  // capture the signed out events
```

Comment:
Can we extract out all these messages in a single file of constants form there we can export the complete object.

```js
const posthogEvents = {
   logout: "User Signed Out",
   login: "User logged in",
}
```

Reply:
Hey @Shruti3004 
Thanks for pointing this out, i have moved all the posthog events to constants